From eb301724da1d378826ad63414a0428c04dae1636 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Sat, 4 May 2019 06:56:41 +0900
Subject: [PATCH] spp_mirror: update makefile for moved common files

As common files are moved from `src/vf/common`, update include and
source paths.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/mirror/Makefile | 17 ++++++++++++-----
 1 file changed, 12 insertions(+), 5 deletions(-)

diff --git a/src/mirror/Makefile b/src/mirror/Makefile
index f1995fc..be58e3a 100644
--- a/src/mirror/Makefile
+++ b/src/mirror/Makefile
@@ -11,21 +11,28 @@ include $(RTE_SDK)/mk/rte.vars.mk
 # binary name
 APP = spp_mirror
 
+SPP_WKT_DIR = ../shared/secondary/spp_worker_th
+
 # all source are stored in SRCS-y
 SRCS-y := spp_mirror.c
 SRCS-y += ../shared/common.c
 SRCS-y += ../shared/secondary/utils.c ../shared/secondary/add_port.c
-SRCS-y += ../vf/common/spp_proc.c ../vf/common/spp_port.c
-SRCS-y += ../vf/common/command_conn.c ../vf/common/command_proc.c
-SRCS-y += ../vf/common/command_dec.c
-SRCS-y += ../vf/common/ringlatencystats.c ../vf/common/string_buffer.c
+SRCS-y += $(SPP_WKT_DIR)/spp_proc.c
+SRCS-y += $(SPP_WKT_DIR)/spp_port.c
+SRCS-y += $(SPP_WKT_DIR)/command_conn.c
+SRCS-y += $(SPP_WKT_DIR)/command_dec.c
+SRCS-y += $(SPP_WKT_DIR)/command_proc.c
+SRCS-y += $(SPP_WKT_DIR)/string_buffer.c
+SRCS-y += $(SPP_WKT_DIR)/ringlatencystats.c
 
 CFLAGS += -DALLOW_EXPERIMENTAL_API
 CFLAGS += $(WERROR_FLAGS) -O3 -MMD
 CFLAGS += -I$(SRCDIR)/../
-CFLAGS += -I$(SRCDIR)/../vf/common
 CFLAGS += -DSPP_MIRROR_MODULE
+
 CFLAGS += -DSPP_MIRROR_SHALLOWCOPY
+
+# Optional Settings
 #CFLAGS += -DSPP_DEMONIZE
 #CFLAGS += -DSPP_RINGLATENCYSTATS_ENABLE
 
-- 
2.17.1

